src/2014/5e-SRD-Feats.json

```diff
@@ -1,4 +1,130 @@
 [
+  {
+    "index": "actor",
```

Comment:
Many of these feats are not in the SRD, so we cannot include them. Check our FAQ for more info about the SRD.

The only Feat in the 2014 SRD is "Grappler", and SRD 5.2 (2024) includes the origin feats Alert, Magic Initiate, Savage Attacker, and the general feat Grappler.